src/containers/Tablet/Tablet.tsx

```diff
@@ -0,0 +1,115 @@
+import {Icon} from '../../components/Icon';
+import {EmptyState} from '../../components/EmptyState';
+import {Loader} from '../../components/Loader';
+
+import {TabletTable} from './TabletTable';
+import {TabletInfo} from './TabletInfo';
+import {TabletControls} from './TabletControls';
+
+import i18n from './i18n';
+
+import './Tablet.scss';
+
+export const b = cn('tablet-page');
+
+export const Tablet = () => {
+    const isFirstDataFetchRef = useRef(true);
+
+    const dispatch = useDispatch();
+
+    const params = useParams<{id: string}>();
+    const {id} = params;
+
+    const {
+        data: tablet = {},
+        loading,
+        id: tabletId,
+        history = [],
+        tenantPath,
+    } = useTypedSelector((state) => state.tablet);
+
+    useEffect(() => {
+        if (isFirstDataFetchRef.current && tablet && tablet.TenantId) {
+            dispatch(getTabletDescribe(tablet.TenantId));
+            isFirstDataFetchRef.current = false;
+        }
+    }, [dispatch, tablet]);
+
+    const fethcData = useCallback(() => {
+        dispatch(getTablet(id));
+    }, [dispatch, id]);
+
+    useAutofetcher(fethcData, [fethcData], true);
+
+    const renderExternalLinks = (link: {name: string; path: string}, index: number) => {
+        return (
+            <li key={index} className={b('link', {external: true})}>
+                <ExternalLink href={`${backend}${link.path}`} target="_blank">
+                    {link.name}
+                </ExternalLink>
+            </li>
+        );
+    };
+
+    if (loading && id !== tabletId && isFirstDataFetchRef.current) {
+        return <Loader size="l" />;
+    }
+
+    if (!tablet || !Object.keys(tablet).length) {
+        return (
+            <div className={b('placeholder')}>
+                <EmptyState title={i18n('emptyState')} />
+            </div>
+        );
+    }
+
+    const {TabletId, Overall, Leader} = tablet;
+
+    const externalLinks = [
+        {
+            name: 'Internal viewer - tablet',
+            path: `/tablets?TabletID=${TabletId}`,
+        },
+    ];
+
+    return (
+        <div className={b()}>
+            <div className={b('pane-wrapper')}>
+                <div className={b('left-pane')}>
+                    <ul className={b('links')}>{externalLinks.map(renderExternalLinks)}</ul>
+                    <div className={b('row', {header: true})}>
+                        <span className={b('title')}>Tablet</span>
+                        <EntityStatus status={Overall} name={TabletId} />
+                        <a
+                            rel="noopener noreferrer"
+                            className={b('link', {external: true})}
+                            href={`${backend}/tablets?TabletID=${TabletId}`}
+                            target="_blank"
+                        >
+                            <Icon name="external" />
+                        </a>
+                        {Leader && <Tag text="Leader" type="blue" />}
```

Comment:
missed translation, but maybe not

Reply:
I didn't translate it intentionally as it corresponds filed in data